DI-1299/compare_b37_and_b38.py

```
@@ -0,0 +1,477 @@
+    Returns
+    -------
+    json_dict : dict
+        the JSON converted to a Python dictionary
+    """
+    with open(file_name, "r", encoding="utf8") as json_file:
+        json_dict = json.load(json_file)
+
+    return json_dict
+
+
+def get_config_info(config_dict):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_config_info`

(ANN201)